package.json

```diff
@@ -78,6 +78,7 @@
   "unpkg": "dist/axios.min.js",
   "typings": "./index.d.ts",
   "dependencies": {
+    "eslint-g": "^1.3.4",
```

Comment:
What is use of this package?

https://www.npmjs.com/package/eslint-g

Reply:
Looks like this shouldn't be here: `To generate eslint config for me.`
Looks like a mistake but having such an unmaintained package with too low downloads and stars in a widely used package (Axios) just scares me.

Reply:
Good call. That was a typo. Apparently, the only purpose of this package is for people to install it by mistake. A very dangerous precedent. It was a typo in a missing space - that's when my IDE required an eslint update. So when you type "npm -i eslint -g" with missing space before -g, you will install this scam. I'm not sure how I missed this change in the commit, but in any case, npm should disallow such names or show an additional prompt to avoid problems like this.